pkg/cmd/server/server.go

```diff
@@ -46,6 +46,9 @@ var ServerCmd = &cobra.Command{
 		})
 
 		views.RenderInfoMessageBold("Starting the Daytona Server daemon...")
+		err = daemon.Stop()
+		if err != nil {
```

Comment:
@ezhil56x this doesn't pass our checks because the `if` is empty.

I suggest that instead of the `if`, you put `log.Trace` here so the error is traced but not displayed to the user unless they set `LOG_LEVEL=trace`.